### PR TITLE
Finalize Round View

### PR DESCRIPTION
**Pull request by boudra**
##### Description

closes https://github.com/gitcoinco/grants-stack/issues/1766

### CODE REVIEW

File: packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx [hunk 1]
```diff
@@ -50,11 +59,78 @@ const distributionOptions = [
...
+        projectId: m.projectId,
+        projectName: m.projectName,
+        contributionsCount: 0,
+        revisedContributionCount: m.contributionsCount,
+        matched: BigNumber.from(m.originalMatchAmountInToken ?? 0).toBigInt(),
+        revisedMatch: BigNumber.from(m.matchAmountInToken ?? 0).toBigInt(),
+      };
+    });
+
+    console.log(distributionMatches, matches);
```
Comment (gravityblast) [inline packages/round-manager/src/features/round/ViewRoundResults/ViewRoundResults.tsx:117]: shall we remove this log?

**Comment by nfrgosselin** [comment]
@boudra the functionality all looks good! The copy doesn't match the design though. More detail in the [Loom](https://www.loom.com/share/f8f5208289e34f2ba63d4c0e1e941800)

**Review by gravityblast — DISMISSED** [review]

**Comment by melissa-neira** [comment]
lgtm, +1 to Nate's comments. One little thing: the hover over the finalize button is red. It makes it seem like something is wrong. Let's get rid of the hover interaction.
Screenshot 2023-05-10 at 16 08 22

**Comment by boudra** [comment]
> 

this PR only updates the finalization results page but I'll fix this here as well

**Review by gravityblast — DISMISSED** [review]

**Review by hmrtn — DISMISSED** [review]

**Review by melissa-neira — APPROVED** [review]

**Review by nfrgosselin — APPROVED** [review]

**Review by gravityblast — APPROVED** [review]

**Review by hmrtn — APPROVED** [review]

**Review by nfrgosselin — APPROVED** [review]

**Review by melissa-neira — APPROVED** [review]

**Pull request merged**